eclcli/bare/v2/server.py

```diff
@@ -548,13 +548,20 @@ def get_parser(self, prog_name):
             metavar="<server>",
             help="Name or ID of server",
         )
+        parser.add_argument(
+            "--type",
+            metavar="<type>",
+            choices=["SOFT", "HARD"],
+            help="Server shutdown mode. A valid value is HARD or SOFT.",
+        )
         return parser
 
     def take_action(self, parsed_args):
         bare_client = self.app.client_manager.bare
 
-        server_obj = utils.find_resource(bare_client.servers,parsed_args.server)
-        bare_client.servers.stop(server_obj.id)
+        body = None if parsed_args.type is None else {'type': parsed_args.type}
+        server_obj = utils.find_resource(bare_client.servers, parsed_args.server)
+        bare_client.servers.stop(server_obj.id, body)
```

Comment:
I don't think body variable is needed.
```suggestion
        bare_client.servers.stop(server_obj.id, parsed_args.type)
```